### PR TITLE
Add Pumpkin (Beta) to Laser Safe List & Add Readme Table of Contents 

### PR DESCRIPTION
**Pull request by caolanxyz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +13,34 @@ This is *not* a configuration management system, but is a handbook for operating
 A 'Meta-service' is an internal definition for 'things that involve lots of services working together' that makes it slightly less confusing to document at a high level... 
 
 This includes things like environmental monitoring, 
+
```
Comment (andrewbolster) [inline README.md:16]: Relative linking plox!

**Review by andrewbolster — CHANGES_REQUESTED** [review]
Ahh, see, the joys of mixed purpose PR's...

Pumpkin settings are bang on, but the readme links need to be updated for relative linking for portability and validation of future forks, or other export formats like `pandoc`ing to PDF for instance. 

Basically drop all the `https://github.com/FarsetLabs/member-handbook/blob/master/` from the links and you should be golden

**Comment by caolanxyz** [comment]
Yeahhhh, intended to file as separate PRs but clearly I haven’t mastered GitHub yet :P 

Will resolve over the weekend.

File: README.md (hunk 1 above)
Comment (caolanxyz) [reply README.md:16]: TIL Relative Linking is a thing (regardless of how obvious the fact...)

Updated now.

**Review by andrewbolster — APPROVED** [review]

**Pull request merged**